fluentd/lib/filter_elasticsearch_genid_ext/lib/filter_elasticsearch_genid_ext.rb

```diff
@@ -0,0 +1,45 @@
+require 'securerandom'
+require 'base64'
+require 'fluent/filter'
+
+module Fluent
+  class ElasticsearchGenidExtFilter < Filter
```

Comment:
Consider actually extending the other filter and calling its filter method if needed.